doc/source/docs/user_guide/mapping.rst

```diff
@@ -180,38 +182,30 @@ Before combining maps, however, remember to always ensure they share a common CR
     # Look at capitals
     # Note use of standard `pyplot` line style options
     @savefig capitals.png
-    cities.plot(marker='*', color='green', markersize=5);
+    groceries.plot(marker='*', color='green', markersize=5);
 
     # Check crs
-    cities = cities.to_crs(world.crs)
+    groceries = groceries.to_crs(chicago.crs)
 
-    # Now you can overlay over country outlines
-    # And yes, there are lots of island capitals
-    # apparently in the middle of the ocean!
+    # Now you can overlay over the outlines
 
 **Method 1**
 
 .. ipython:: python
 
-    base = world.plot(color='white', edgecolor='black')
-    @savefig capitals_over_countries_1.png
-    cities.plot(ax=base, marker='o', color='red', markersize=5);
+    base = chicago.plot(color='white', edgecolor='black')
+    @savefig groceries_over_chicago_1.png
+    groceries.plot(ax=base, marker='o', color='red', markersize=5);
 
 **Method 2: Using matplotlib objects**
 
 .. ipython:: python
 
-    import matplotlib.pyplot as plt
     fig, ax = plt.subplots()
 
-    # set aspect to equal. This is done automatically
-    # when using GeoPandas plot on it's own, but not when
-    # working with pyplot directly.
-    ax.set_aspect('equal')
```

Comment:
Just checking, this note was deliberately removed?

Reply:
Yes, we made changes to aspect settings in the meantime so this is not needed.